examples/k8s-health-check/proxy_with_http_health_check.yaml

```diff
@@ -49,24 +49,18 @@ spec:
       - name: cloud-sql-proxy
         # It is recommended to use the latest version of the Cloud SQL proxy
         # Make sure to update on a regular schedule!
-        image: gcr.io/cloudsql-docker/gce-proxy:1.28.0 # make sure the use the latest version
-        command:
-          - "/cloud_sql_proxy"
-
+        image: gcr.io/cloudsql-docker/gce-proxy:2.0.0 # make sure the use the latest version
```

Comment:
This will be `gcr.io/cloud-sql-connectors/cloud-sql-proxy:2.0.0.preview.0` after https://github.com/GoogleCloudPlatform/cloudsql-proxy/pull/1326 is merged.

Reply:
Ditto below.

Reply:
I updated all occurrences to gcr.io/cloud-sql-connectors/cloud-sql-proxy:2.0.0.preview.0